app/models/region.rb

```diff
@@ -54,6 +54,36 @@ def log_payload
     attrs.symbolize_keys
   end
 
+  def syncable_patients
+    case region_type
+      when "block"
+        registered_patients
+          .union(assigned_patients)
+          .union(appointed_patients)
+      else
+        registered_patients
+    end
+  end
+
+  def registered_patients
+    Patient
+      .with_discarded
+      .where(registration_facility: facility_regions.pluck(:source_id))
+  end
+
+  def assigned_patients
+    Patient
+      .with_discarded
```

Comment:
Same here, adding `with_discarded` would break RegionReports, as we definitely don't want soft deleted patients there.

See https://github.com/simpledotorg/simple-server/pull/1477/files#diff-7a3a5da2640933d23aba82fb308d4cb9827e42c27ed05ba979cdb9aa6b849637R51

Reply:
Good catch!